pkgs/os-specific/linux/waydroid/default.nix

```diff
@@ -51,7 +51,7 @@ python3Packages.buildPythonApplication rec {
   dontWrapGApps = true;
 
   installPhase = ''
-    make install PREFIX=$out USE_SYSTEMD=0
+    make install DESTDIR=$out PREFIX= USE_SYSTEMD=0
```

Comment:
Are you sure this is correct? Typically DESTDIR doesn't modify the contents of a package to indicate its location in the filesystem. While PREFIX is supposed to set its absolute location, at times in the package's files.

Reply:
Yes, PREFIX defaults to /usr which we don't want.

Reply:
> Yes, PREFIX defaults to /usr which we don't want.

That's why we set it to $out, see also https://github.com/NixOS/nixpkgs/issues/65718

Reply:
ammended the fix

Reply:
Can someone help me with #1218 ?